expression/bench_test.go

```diff
@@ -643,7 +643,7 @@ func genVecExprBenchCase(ctx sessionctx.Context, funcName string, testCase vecEx
 		panic(err)
 	}
 
-	output = chunk.New([]*types.FieldType{eType2FieldType(testCase.retEvalType)}, 1024, 1024)
+	output = chunk.New([]*types.FieldType{eType2FieldType(expr.GetType().EvalType())}, 1024, 1024)
```

Comment:
After investigating, we found this is the root cause resulting in the CI problem when setting the `RetType` of `DayName` to `Int` or `Real`.
The actual `RetType` of this `expr` has been modified to `String` but we prepare an `Int/Real` column to store its data, which causes panic when accessing `Column.offset`.

Only in the test function `TestVectorizedBuiltinTimeEvalOneVec`, the `RetType` will be modified, while in another function `TestVectorizedBuiltinTimeFunc`, the `RetType` is still `Int`, so these test cases whose `RetType` is `Int` or `Real` work.

PTAL @SunRunAway